infoschema/tables_test.go

```diff
@@ -606,7 +606,34 @@ func prepareSlowLogfile(c *C, slowLogFileName string) {
 # Plan: abcd
 # Plan_digest: 60e9378c746d9a2be1c791047e008967cf252eb6de9167ad3aa6098fa2d523f4
 # Prev_stmt: update t set i = 2;
-select * from t_slim;`))
+select * from t_slim;
+# Time: 2021-09-08T14:39:54.506967433+08:00
+# Txn_start_ts: 427578666238083075
+# User@Host: root[root] @ 172.16.0.0 [172.16.0.0]
+# Conn_ID: 40507
+# Query_time: 25.571605962
+# Parse_time: 0.002923536
+# Compile_time: 0.006800973
+# Rewrite_time: 0.002100764
+# Optimize_time: 0
+# Wait_TS: 0.000015801
+# Prewrite_time: 25.542014572 Commit_time: 0.002294647 Get_commit_ts_time: 0.000605473 Commit_backoff_time: 12.483 Backoff_types: [tikvRPC,regionMiss,tikvRPC,regionMiss,tikvRPC,regionMiss,tikvRPC,regionMiss,tikvRPC,regionMiss,tikvRPC,regionMiss,tikvRPC,regionMiss,tikvRPC,regionMiss,tikvRPC,regionMiss,tikvRPC,regionMiss] Write_keys: 624 Write_size: 172064 Prewrite_region: 60
```

Comment:
```sql
Backoff_types: [tikvRPC regionMiss]
```

For the old version of slow log, the parse will still be wrong, I think we should fix this too.

Reply:
It sounds reasonable, but is there really happen that the new parser working on the old version of slow log? 🤔

Reply:
It will happen, after TiDB upgrades to the new version, it still can parse the old slow log. The old slow log will not be removed unless the human manual deletes it.

Reply:
Sounds like we can match `xxxx:` to distinguish fields.

IMO, we may also need some regression tests to verify that the slow log can be correctly parsed by percona tools.

Reply:
match `xxxx:` to distinguish fields is a great idea. I like it.

Reply:
> Sounds like we can match `xxxx:` to distinguish fields.
> 
> IMO, we may also need some regression tests to verify that the slow log can be correctly parsed by percona tools.

LGTM, I will add this pattern.

Reply:
I made another solution about this , but I am not sure if it is better since it used `regex`.

Reply:
What if we make both parser exits, when ',' exits, we used the new parser. To old version, we use the regex parser with little perf loss.🤔

Reply:
I think about the solution once again. I prefer the `comma added` solution, reasons are like that:
1. clear format makes it easy to parse by other tools.
2. compatibility problem above is a small probability event, and even it happens, we can easily solve it by `flush logs`.
3. slow log can rotate by the default config, which makes the little bug disappear when time goes on.